deploy-board/deploy_board/templates/hosts/hosts.tmpl

```diff
@@ -147,9 +152,9 @@
         });
 
         if ($("#hostTable input:checkbox:checked").length <= 0) {
-            $(".btn").prop("disabled", true);
+            $(".checked-hosts-btn").prop("disabled", true);
         } else {
-            $(".btn").prop("disabled", false);
+            $(".checked-hosts-btn").prop("disabled", false);
         }
```

Comment:
Can one of these be removed? 
```
if ($("#hostTable input:checkbox:checked").length <= 0) {
            $(".checked-hosts-btn").prop("disabled", true);
        } else {
            $(".checked-hosts-btn").prop("disabled", false);
        }
```

Reply:
As far as we don't have any buttons which can be clicked only by checked hosts, I deleted this JQuery code to enable/disable buttons.

Reply:
The Pause, Resume etc. buttons on the upper right corner still need to disabled when there is no host checked, right?

Reply:
Oh, yes. I fixed this.  Also we can't delete code from your comment https://github.com/pinterest/teletraan/pull/1408#discussion_r1459469501 because this code is responsible for enabling and disabling buttons.

<img width="1408" alt="Screenshot 2024-01-19 at 4 51 55 PM" src="https://github.com/pinterest/teletraan/assets/20383875/225978e6-ca42-4912-b2e2-1af13c46b241">

Reply:
LG. My original question is regarding whether one of the repeated listener can be removed.

Reply:
No, them are responsible for different things:
this listener https://github.com/pinterest/teletraan/pull/1408/files/0cc10746727c3be9870601e0d788bedec37c14d0#diff-9311ec701ea3da55954c03dcb3ad085f0319ea31328bc44bab0319cd33f26915R130-R135 
is responsible for enable and disable button after user click.

this listener
https://github.com/pinterest/teletraan/pull/1408/files/0cc10746727c3be9870601e0d788bedec37c14d0#diff-9311ec701ea3da55954c03dcb3ad085f0319ea31328bc44bab0319cd33f26915L149-L153

is responsible for initially render button on the first page load. So we can't remove any of them.

Reply:
This is not repeated listeners, them are different. Of course we can refactor code and create a separated function for it, but again as for me this is out of scope of current ticket.

Reply:
What about
```html
<button id="resumeButton"
          class="btn btn-default btn-sm checked-hosts-btn" data-toggle="tooltip"
          title="Resume selected hosts" disabled>
```
mark as resolved